code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -246,6 +247,7 @@
 	if(callback)
 		callback.Remove(C)
 	C.faction -= "slime"
+	C.blood_volume = min(C.blood_volume, BLOOD_VOLUME_NORMAL)
```

Comment:
slime people blood volume can go super high (to split into two), so we reduce it to normal level if necessary on species change.